docs/topic-guides/hp-tuning-det/hp-pbt.txt

```diff
@@ -73,8 +73,7 @@ fixed amount of training, determined by the experiment configuration.
 -  ``population_size``: The number of trials that should run at the same
    time.
 -  ``num_rounds``: The total number of rounds to run.
--  ``steps_per_round``: The number of training steps for each trial to
-   run during each round.
+-  ``length_per_round``: The training length for each trial during a round.
```

Comment:
Seems a bit confusing what the units are here -- is this number of epochs, number of minibatches, or both/either?

Reply:
Addressing this and all similar comments by adding a subsection describing this nested configuration.

It is records, batches or epochs in all cases.